hexrdgui/pinhole_correction_editor.py

```diff
@@ -475,18 +438,15 @@ def apply_panel_buffer_visible(self, b):
 
     def auto_select_rygg_absorption_length(self):
         w = self.ui.rygg_absorption_length_selector
-        value = self.ui.rygg_absorption_length_value.value()
 
-        # Check if there is an absorption length that matches other than
-        # "Enter Manually", and auto set it if there is.
+        # Set the material to match the pinhole package if it exists.
+        # If not default to "Enter Manually"
         for i in range(1, w.count()):
             name = w.itemText(i)
             if not name:
                 # It's a separator
                 continue
-
-            absorption_length = self.get_rygg_absorption_length(i)
-            if np.isclose(value, absorption_length):
+            if name == HexrdConfig().pinhole_package.material:
```

Comment:
Are we still letting users manually edit the absorption correction length in the Rygg pinhole correction editor? If not, I don't think we need this, right?